src/NLog/NewRelic.LogEnrichers.NLog/NewRelic.LogEnrichers.NLog.csproj

```diff
@@ -30,6 +30,8 @@
     <AssemblyVersion>$(Major).0.0.0</AssemblyVersion>
     <FileVersion>$(Major).$(Minor).$(Build).$(Revision)</FileVersion>
 
+    <GitTagPrefix>NewRelic.LogEnrichers.NLog_v</GitTagPrefix>
```

Comment:
We may want to simplify the tag prefix to just NLog_v and use a similar shortened prefix for serilog.

Reply:
Cool by me. Does make it easier on the eyes.

Reply:
I think that it makes it easier. It limits it to the most important information, and it's also less that we need to specify when creating a tag, so hopefully that means less room for manual error.